src/Components/Header/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./Header.scss";
+import { Link } from "react-router-dom";
+
+const Header = () => {
+
+  return (
+    <header>
+        <img className="logo"  src="./assets/images/logo-gray@3x.png" alt="uçurtma" />
```

Comment:
can we write `Uçurtma` instead of `uçurtma`?

also, there is a one unnecessary space between `className` and `src`. normally, eslint should warn you about them but i guess you aren't have eslint and prettier extension in your editor.